src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// Action Creators
+export const addBook = (book) => ({ type: ADD_BOOK, payload: book });
+export const removeBook = (id) => ({ type: REMOVE_BOOK, payload: { id } });
```

Comment:
- [ ] Is there really any need to further destructure the `id` here? Since we are passing the book id directly as a parameter, we can change the expression to something like so `..., payload: id`